docs/production-deployment/cloud/audit-logging.mdx

```diff
@@ -95,132 +94,49 @@ The log sent to the Kinesis stream is JSON in the following format:
 **********
-{"emit_time":"2023-11-08T08:14:09Z","level":"LOG_LEVEL_INFO","user_email":"user1@example.com","operation":"UpdateNamespace","details":{"namespace":"audit-log-test.example-dev","client_ca_fingerprints":["f186d0bd971ff7d52dc6cc9d9b6f7644"],"search_attribute_update":{}},"status":"OK","category":"LOG_CATEGORY_ADMIN","log_id":"0mc69c0323b871293ce231dd1c7fb638","principal":{"id":"988cb80b-d6be-4bb5-9c87-d09f93f58ed3","type":"user","name":"user1@example.com"}}
+{"emit_time":"2023-11-08T08:14:09Z","level":"LOG_LEVEL_INFO","caller_ip_address":"10.1.2.3, 10.4.5.6","user_email":"user1@example.com","operation":"UpdateNamespace","details":{"namespace":"audit-log-test.example-dev","client_ca_fingerprints":["f186d0bd971ff7d52dc6cc9d9b6f7644"],"search_attribute_update":{}},"status":"OK","category":"LOG_CATEGORY_ADMIN","log_id":"0mc69c0323b871293ce231dd1c7fb638","principal":{"id":"988cb80b-d6be-4bb5-9c87-d09f93f58ed3","type":"user","name":"user1@example.com"}}
 **********
-{"emit_time":"2023-11-08T09:20:22Z","level":"LOG_LEVEL_INFO","user_email":"user1@example.com","operation":"UpdateUserNamespacePermissions","details":{"namespace":"audit-log-test.example-dev","search_attribute_update":{}},"status":"OK","category":"LOG_CATEGORY_ADMIN","log_id":"0mc69c0323b871293ce231dd1c7fb639","principal":{"id":"988cb80b-d6be-4bb5-9c87-d09f93f58ed3","type":"user","name":"user1@example.com"}}
+{"emit_time":"2023-11-08T09:20:22Z","level":"LOG_LEVEL_INFO","caller_ip_address":"10.1.2.3, 10.4.5.6","user_email":"user1@example.com","operation":"UpdateUserNamespacePermissions","details":{"namespace":"audit-log-test.example-dev","search_attribute_update":{}},"status":"OK","category":"LOG_CATEGORY_ADMIN","log_id":"0mc69c0323b871293ce231dd1c7fb639","principal":{"id":"988cb80b-d6be-4bb5-9c87-d09f93f58ed3","type":"user","name":"user1@example.com"}}
 **********
 ```
 
-### Example of consuming an Audit Log
-
-The following Go code is an example of consuming Audit Logs from a Kinesis stream and delivering them to an S3 bucket.
-
-```go
-func main() {
-   fmt.Println("print audit log from S3")
-   cfg, err := config.LoadDefaultConfig(context.TODO(),
-      config.WithSharedConfigProfile("your_profile"),
-   )
-   if err != nil {
-      fmt.Println(err)
-   }
-   s3Client := s3.NewFromConfig(cfg)
-   response, err := s3Client.GetObject(
-      context.Background(),
-      &s3.GetObjectInput{
-         Bucket: aws.String("your_bucket_name"),
-         Key:    aws.String("your_s3_file_path")})
-   if err != nil {
-      fmt.Println(err)
-   }
-   defer response.Body.Close()
-
-   content, err := io.ReadAll(response.Body)
-
-   fmt.Println(string(content))
-}
-```
+## How to configure Audit Logging {#configure-audit-logging}
 
-The preceding code also prints the logs in the terminal.
-The following is a sample result.
+Audit logging can be configured in both AWS Kinesis
+
+* [AWS Kinesis Instructions](/cloud/audit-logging-aws)
+
+## How to consume an Audit Log {#consume-an-audit-log}
```

Comment:
Let's change this to Consume an Audit Log and under it "How to consume an Audit Log" - this is better for SEO and Kapa purposes

Reply:
Done.